cli/mysql.go

```diff
@@ -0,0 +1,185 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/cheggaaa/pb"
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/docker/docker/pkg/term"
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/flynn/go-docopt"
+	"github.com/flynn/flynn/controller/client"
+	ct "github.com/flynn/flynn/controller/types"
+)
+
+func init() {
+	register("mysql", runMysql, `
+usage: flynn mysql console [--] [<argument>...]
+       flynn mysql dump [-q] [-f <file>]
+       flynn mysql restore [-q] [-f <file>]
+
+Options:
+	-f, --file=<file>  name of dump file
+	-q, --quiet        don't print progress
+
+Commands:
+	console  Open a console to a Flynn mysql database. Any valid arguments to mysql may be provided.
```

Comment:
For the CLI, `mysql` makes sense to me since regardless of the implementation used it would likely go through the same CLI.